### DISCUSSION TITLE
Need help. Tried to update and system stuck on I initializing navigation at 11%

### DISCUSSION THREAD

**Discussion by bwilliams75** [Mr-MIBonk/M.I.B._More-Incredible-Bash#538] [Q&A]
Title says it all. I have a 2018 sq5 and I was trying to update for the unlimited navigation update. I followed all the instructions to a t and everything was running well until the last restart. The system got to initializing navigation and gets to 11% and then hangs.  It will restart itself after a while and repeat. I can’t enter either the green or red menus. I have tried disconnecting the battery and rebooting the system and nothing works. I did make a back up through MIB and at this point I would be happy with just being able to restore that. 

If anyone out there can help, please let me know.

**Comment by Tomt97** [comment]
this this one out.. https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/discussions/448
Just got mine fixed.
